src/kOS.Safe/Encapsulation/ListValue.cs

```diff
@@ -297,7 +297,7 @@ private void InitializeSuffixes()
             AddSuffix("COPY", new NoArgsSuffix<ListValue>(() => new ListValue(this)));
         }
 
-        public static ListValue CreateList<T>(IList<T> toCopy)
+        public new static ListValue CreateList<T>(IEnumerable<T> toCopy)
```

Comment:
The two list value classes had different signatures for this method. this reconciles them.